packages/sku/lib/parseArgs.js

```diff
@@ -19,24 +17,10 @@ module.exports = (processArgv) => {
     // We need the first arg we give to minimist to be the script name.
     processArgv.slice(2),
     {
-      string: [
-        'env',
-        'tenant',
-        'build',
-        'config',
-        'environment',
-        'packageManager',
-        'site',
-        'stats',
-      ],
-      default: {
-        env: 'production',
-        tenant: '',
-      },
+      string: ['config', 'environment', 'packageManager', 'site', 'stats'],
       alias: {
-        e: 'env',
+        e: 'environment',
         t: 'tenant',
```

Comment:
This should go too right?

Reply:
https://github.com/seek-oss/sku/pull/980/commits/1ced58d3a44487409ddf70eb8674c397f1ec65ed